tests/src/test/scala/system/basic/WskBasicTests.scala

```diff
@@ -82,10 +84,10 @@ class WskBasicTests extends TestHelpers with WskTestHelpers {
       pkg.create(name, update = true)
     }
     val stdout = wsk.pkg.get(name).stdout
-    stdout should include regex (""""key": "a"""")
-    stdout should include regex (""""value": "A"""")
-    stdout should include regex (""""publish": true""")
-    stdout should include regex (""""version": "0.0.2"""")
+    stdout should include regex (""""key":"a"""")
+    stdout should include regex (""""value":"A"""")
+    stdout should include regex (""""publish":true""")
+    stdout should include regex (""""version":"0.0.2"""")
```

Comment:
Can we parse the JSON response of the API and then assert on the parsed values? That would remove those whitespace issues alltogether.